app/javascript/controllers/keyboardable_controller.js

```diff
@@ -7,6 +7,6 @@ export default class extends Controller {
 
   navigate() {
     const new_conversation_path = this.pathValue
-    Turbo.visit(new_conversation_path) 
+    Turbo.visit(new_conversation_path)
```

Comment:
@florianfelsing I didn't mean to make this change, but my code editor strips trailing whitespace

Reply:
Mine, too, usually. I probably messed something up in my JS auto-formatting config when I made some modification last week. Thanks for pointing this out!